components/aws/sagemaker/tests/integration_tests/scripts/run_integration_tests

```diff
@@ -17,7 +17,7 @@ REGION=${REGION:-"$(aws configure get region)"} # Deployment region
 
 ### Configuration parameters
 EKS_EXISTING_CLUSTER=${EKS_EXISTING_CLUSTER:-""} # Use an existing EKS cluster
-EKS_CLUSTER_VERSION=${EKS_CLUSTER_VERSION:-"1.19"} # EKS cluster K8s version
+EKS_CLUSTER_VERSION=${EKS_CLUSTER_VERSION:-"1.20"} # EKS cluster K8s version
```

Comment:
change to 1.21?

Reply:
can do. we had once discussed going one version at a time.